### PR TITLE
Antonsekatski/add uuid example

### PR DESCRIPTION
**Pull request by antonsekatski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,15 +247,24 @@ Returned promise rejects with an error if there is no previously authenticated u
...
 
 ``` bash
+cd examples
+
 # install dependencies
 npm install
 
 # serve examples with hot reload at localhost:8080
+npm run server # start API server for UUID examples
 npm run dev
```
Comment (kesor) [inline README.md]: Maybe use node-foreman here? A single Procfile takes care of running both servers. There is already a Procfile in this `examples/` folder, just mention it in the readme and make it run both servers.

Probably also a good idea to mention that both of these services will be executed, and these will be using the `.env` file for configuration.

File: examples/server/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import express from 'express';
+import AWS from 'aws-sdk';
+import bodyParser from 'body-parser';
+
+const app = express();
+
+app.use((req, res, next) => {
```
Comment (kesor) [inline examples/server/index.js]: Useful to add at least one comment line saying that this is for `CORS`.

File: examples/server/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  res.type('application/json; charset=utf-8');
+  next();
+});
+
+// JSON body parsing
+app.use(bodyParser.json());
+
+const cognito = new AWS.CognitoIdentityServiceProvider({ apiVersion: '2016-04-19', region: 'us-east-1' });
```
Comment (kesor) [inline examples/server/index.js]: The region should probably also be part of the `.env` configuration.

Comment (antonsekatski) [inline examples/server/index.js]: Not quite sure about apiVersion here. I think it couldn't be added in .env file.

Comment (kesor) [inline examples/server/index.js]: The API version doesn't change, this is how AWS keep their APIs backwards compatible. Usually you can just omit this and the SDK will choose the latest version.

**Pull request merged**